### PR TITLE
Added `socket.dynamic` and `socket.ssl_reload` parameter description

### PR DESCRIPTION
**Pull request by mkaruza**

### CODE REVIEW

File: galeracluster/source/documentation/galera-parameters.rst [hunk 1]
```diff
@@ -157,10 +157,12 @@ Below is a list of all of the Galera parameters.  Each is also a link to further
    ":ref:`socket.ssl_ca <socket.ssl_ca>`", "", "  No", "", "1.0"
    ":ref:`socket.ssl_cert <socket.ssl_cert>`", "", "  No", "", "1.0"
    ":ref:`socket.checksum <socket.checksum>`", "``1`` (vs. 2); ``2`` (vs. 3)", "  No", "", "2.0"
+   ":ref:`socket.dynamic <socket.dynamic>`", "FALSE", "  No", "", "4.0"
```
Comment (temeo) [inline galeracluster/source/documentation/galera-parameters.rst:160]: I think initial version field should be 4.8 for both of these variables, change also below in descriptions.

Comment (mkaruza) [reply galeracluster/source/documentation/galera-parameters.rst]: Changed.

File: galeracluster/source/documentation/galera-parameters.rst [hunk 2]
```diff
@@ -2001,6 +2003,33 @@ The excerpt below is an example of how this Galera parameter might look in the c
...
+Enable connection engine to accept both SSL and TCP connections.   
+
+.. csv-table::
+   :class: doc-options
+
+   "Default Value", "false"
+   "Dynamic", "No"
+   "Initial Version", "4.8"
+
+By enabling this parameter, Galera should be possible to communicate with both SSL and TCP connections.
```
Comment (byte) [inline galeracluster/source/documentation/galera-parameters.rst:2022]: By enabling this parameter, it should be possible for Galera to communicate with both SSL and TCP connections.

**Review by byte — CHANGES_REQUESTED** [review]
Minus the one line change I requested, this is ok to push

File: galeracluster/source/documentation/galera-parameters.rst (hunk 2 above)
Comment (mkaruza) [reply galeracluster/source/documentation/galera-parameters.rst]: Changed.

**Review by byte — APPROVED** [review]
Looks good, thanks @mkaruza

**Pull request merged**